utils/config.go

```diff
@@ -0,0 +1,50 @@
+package utils
+
+import (
+  "fmt"
+  "os"
+  "time"
+)
+
+const (
+  // DefaultKhaosInterval is the default time in seconds between khaos events
+  DefaultKhaosInterval = 30
+)
+
+type Config struct {
+  Name string
+  Namespace string
+  KhaosInterval time.Duration
+}
+
+func NewConfig() (conf *Config, err error) {
+  conf = &Config{}
+
+  // get namespace to wreak havoc in
+  var namespace string
+  if namespace = os.Getenv("KHAOS_NAMESPACE"); namespace == "" {
+    // shouldn't error our b.c we are using Downward API, but better safe than sorry
+    return nil, fmt.Errorf("No namespace available in environment")
+  }
+
+  conf.Namespace = namespace
+
+  // get self name so as to avoid suicide
+  var name string
+  if name = os.Getenv("KHAOS_MONKEY_NAME"); name == "" {
```

Comment:
It's worth noting that this requires configuration details to be setup to use the downward API properly. Assuming khaos monkey is deployed as a `deployment` object it will have a hash in its `pod` name so this can't be a static value. Would be nice if there was a way to do this with fewer configuration requirements. Just a nit though.